pkg/controller/framework/controller_test.go

```diff
@@ -113,3 +116,272 @@ func Example() {
 	// b-controller
 	// c-framework
 }
+
+func ExampleInformer() {
+	// source simulates an apiserver object endpoint.
+	source := framework.NewFakeControllerSource()
+
+	// Let's do threadsafe output to get predictable test results.
+	outputSetLock := sync.Mutex{}
+	outputSet := util.StringSet{}
+
+	// Make a controller that immediately deletes anything added to it, and
```

Comment:
I take it that the call to source.Delete() in AddFunc() will trigger DeleteFunc() ?